webapp/src/app/files/file.component.html

```diff
@@ -0,0 +1,167 @@
+<!-- FILES -->
+<app-card icon="files-o" title="Files" [createButton]="fileService.getCardButtons()" (action)="create()">
+  <div class="row">
+    <div class="col-12 col-lg-8">
+      <table class="table table-striped table-sm">
+        <thead>
+          <tr>
+            <th *ngFor="let header of fileService.getTableHeaders()">{{ header }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr *ngFor="let item of fileService.filesToContainers">
+            <td>{{ item.container.name }}</td>
+            <td>
+              <div *ngFor="let file of item.files">
+                <div class="row">
+                  <div class="col-2">
+                    <button *ngIf="file.ext === 'wav'" type="button" class="btn btn-sm btn-primary" (click)="play(file)">
```

Comment:
We should probably get some default icon in in case the uploaded file is not a `.wav`